sdk/dotnet/Dynamodb/GetTable.cs

```diff
@@ -15,25 +16,25 @@ public static partial class Invokes
         /// &gt; This content is derived from https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/d/dynamodb_table.html.markdown.
         /// </summary>
         public static Task<GetTableResult> GetTable(GetTableArgs args, InvokeOptions? options = null)
-            => Pulumi.Deployment.Instance.InvokeAsync<GetTableResult>("aws:dynamodb/getTable:getTable", args ?? ResourceArgs.Empty, options.WithVersion());
+            => Pulumi.Deployment.Instance.InvokeAsync<GetTableResult>("aws:dynamodb/getTable:getTable", args ?? InvokeArgs.Empty, options.WithVersion());
     }
 
-    public sealed class GetTableArgs : Pulumi.ResourceArgs
+    public sealed class GetTableArgs : Pulumi.InvokeArgs
     {
         /// <summary>
         /// The name of the DynamoDB table.
         /// </summary>
         [Input("name", required: true)]
-        public Input<string> Name { get; set; } = null!;
+        public string Name { get; set; } = null!;
 
         [Input("serverSideEncryption")]
-        public Input<Inputs.GetTableServerSideEncryptionArgs>? ServerSideEncryption { get; set; }
+        public Inputs.GetTableServerSideEncryptionArgs? ServerSideEncryption { get; set; }
```

Comment:
This is likely to fail, so we might want to hold off until https://github.com/pulumi/pulumi/pull/3614 is released